app/src/main/java/org/mozilla/focus/session/ui/SessionViewHolder.kt

```diff
@@ -40,24 +38,10 @@ class SessionViewHolder internal constructor(
         updateUrl(session)
 
         val isCurrentSession = SessionManager.getInstance().isCurrentSession(session)
-        val actionColor = ContextCompat.getColor(textView.context, R.color.colorAction)
-        val darkColor = ContextCompat.getColor(textView.context, R.color.colorSession)
-
-        updateTextColor(isCurrentSession, actionColor, darkColor)
-        updateDrawable(isCurrentSession, actionColor, darkColor)
-    }
-
-    private fun updateTextColor(isCurrentSession: Boolean, actionColor: Int, darkColor: Int) {
-        textView.setTextColor(if (isCurrentSession) actionColor else darkColor)
-    }
-
-    private fun updateDrawable(isCurrentSession: Boolean, actionColor: Int, darkColor: Int) {
-        val drawable = AppCompatResources.getDrawable(itemView.context, R.drawable.ic_link) ?: return
-
-        val wrapDrawable = DrawableCompat.wrap(drawable.mutate())
-        DrawableCompat.setTint(wrapDrawable, if (isCurrentSession) actionColor else darkColor)
-
-        textView.setCompoundDrawablesWithIntrinsicBounds(wrapDrawable, null, null, null)
+        val color = Color.rgb(53, 56, 82)
+        if(isCurrentSession) {
```

Comment:
\[ktlint\] WARNING: \[keyword\-spacing\] Missing spacing after "if" in column 11